fuse_models/include/fuse_models/unicycle_2d_state_cost_function.h

```diff
@@ -182,6 +185,8 @@ class Unicycle2DStateCostFunction : public ceres::SizedCostFunction<8, 2, 1, 2,
       if (jacobians[0])
       {
         Eigen::Map<fuse_core::Matrix<double, 8, 2>> jacobian(jacobians[0]);
+        const fuse_core::Matrix2d R_yaw_inv = fuse_core::rotationMatrix2D(-parameters[1][0]);
+        jacobian.block<2, 2>(0, 0) = R_yaw_inv * jacobian.block<2, 2>(0, 0);
```

Comment:
Mostly curious, but I wonder if you can do this
```suggestion
        jacobian.block<2, 2>(0, 0).applyOnTheLeft(R_yaw_inv);
```

I also wonder if the sign is correct, since we have `- position1`

Reply:
it should be, we multiply by -A for position1 (and all "1" variables), and we use positive A for "2" variables